profile/src/v100/entity/persona/persona_data/entry_kinds/phone_number.rs

```diff
@@ -0,0 +1,121 @@
+use crate::prelude::*;
+
+#[derive(
+    Serialize,
+    Deserialize,
+    Clone,
+    PartialEq,
+    Hash,
+    Eq,
+    derive_more::Display,
+    derive_more::Debug,
+    uniffi::Record,
+)]
+#[display("{number}")]
+#[debug("{number}")]
+#[serde(transparent)]
+pub struct PhoneNumber {
```

Comment:
reconsider macro merging `PhoneNumber` and `EmailAddress` (and possible `DisplayName`)???

Reply:
hmm I deem it just not worth it. In future we might wanna add different validation, and also it makes documentation a bit harder since harder to document macro defined structs.